package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_bizarre_cravings",
+  "version": "1.0.0",
+  "description": "Hello",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
```

Comment:
Add "dependencies" and "devDependencies" fields to manage your project dependencies. [important]